artifact/local/fs.go

```diff
@@ -13,6 +13,8 @@ import (
 	"golang.org/x/xerrors"
 
 	"github.com/aquasecurity/fanal/analyzer"
+	_ "github.com/aquasecurity/fanal/analyzer/os/alpine"
```

Comment:
NIT: We have these imported in cmd/fanal/main.go ; I guess you added the imports for side effects to make test runner happy?

Reply:
Absolutely. I realized these import should be in fs_test.go and moved them in the previous PR. But I reverted it and they are in fs.go again. I'll fix it in the dedicated PR.